### ISSUE TITLE
tensorflow 1.14 not picking up GPU: CUBLAS_STATUS_INTERNAL_ERROR

### ISSUE THREAD

**Issue by ktattan** [tensorflow/tensorflow#35064]
I am having problems running tensorflow on my GPU.

My environment is as follows:

* OS: Linux Mint
* GPU: GeForce RTX 2070 Super
* Nvidia Driver Version: 435.21
* CUDA Version: 10.1
* gcc --version: gcc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
* tensorflow-gpu 1.14
* keras 2.3.1

Both the keras and tensorflow packages are located in a conda environment with no other tensorflow or keras versions to avoid package conflict. First, I get conflicting GPU devices when I run the following quick test:

```
# picks up the GPU it seems
from tensorflow.python.client import device_lib
print(device_lib.list_local_devices())
>>> [name: "/device:CPU:0"
device_type: "CPU"
memory_limit: 268435456
locality {
}
incarnation: 4346857393168915334
, name: "/device:XLA_CPU:0"
device_type: "XLA_CPU"
memory_limit: 17179869184
locality {
}
incarnation: 15716071101553989809
physical_device_desc: "device: XLA_CPU device"
, name: "/device:XLA_GPU:0"
device_type: "XLA_GPU"
memory_limit: 17179869184
locality {
}
incarnation: 6257014534476475142
physical_device_desc: "device: XLA_GPU device"
]

# but then Keras doesn't pick up the GPU?
from keras import backend as K
K.tensorflow_backend._get_available_gpus()
>>> []
```
This doesn't make much sense as I thought keras uses tensorflow as the backend.

Then I tried running the below program (I got some hints from these two SO posts [[1]](https://datascience.stackexchange.com/a/41958/41929)[[2]](https://stackoverflow.com/a/52132342/4139143))

```
import numpy as np
import tensorflow as tf
from datetime import datetime
import warnings
warnings.filterwarnings("ignore")

# Choose which device you want to test on: either 'cpu' or 'gpu'
devices = ['/device:CPU:0', '/device:XLA_GPU:0']

# Choose size of the matrix to be used.
# Make it bigger to see bigger benefits of parallel computation
shapes = [(50, 50), (100, 100), (500, 500), (1000, 1000)]


def compute_operations(device, shape):
    """Run a simple set of operations on a matrix of given shape on given device

    Parameters
    ----------
    device : the type of device to use, either 'cpu' or 'gpu'
    shape : a tuple for the shape of a 2d tensor, e.g. (10, 10)

    Returns
    -------
    out : results of the operations as the time taken
    """

    # Define operations to be computed on selected device
    with tf.device(device):
        random_matrix = tf.random.uniform(shape=shape, minval=0, maxval=1)
        dot_operation = tf.matmul(random_matrix, tf.transpose(random_matrix))
        sum_operation = tf.reduce_sum(dot_operation)

    # Time the actual runtime of the operations
    config = tf.ConfigProto()
    config.gpu_options.allow_growth = True
    start_time = datetime.now()
    with tf.compat.v1.Session(config=config) as session:
            result = session.run(sum_operation)
    elapsed_time = datetime.now() - start_time

    return result, elapsed_time


if __name__ == '__main__':

    # Run the computations and print summary of each run
    for device in devices:
        print("--" * 20)

        for shape in shapes:
            _, time_taken = compute_operations(device, shape)

            # Print the result and also the time taken on the selected device
            print("Input shape:", shape, "using Device:", device, "took: {:.2f}".format(time_taken.seconds + time_taken.microseconds/1e6))
            #print("Computation on shape:", shape, "using Device:", device, "took:")
```

But got multiple errors
```
2019-12-12 18:35:05.957991: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
name: GeForce RTX 2070 SUPER major: 7 minor: 5 memoryClockRate(GHz): 1.8
pciBusID: 0000:42:00.0
...
...
2019-12-12 18:35:06.655452: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Could not dlopen library 'libcublas.so.10.0'; dlerror: libcublas.so.10.0: cannot open shared object file: No such file or directory
2019-12-12 18:35:06.655474: E tensorflow/stream_executor/cuda/cuda_blas.cc:238] failed to create cublas handle: CUBLAS_STATUS_INTERNAL_ERROR
2019-12-12 18:35:06.655483: F tensorflow/compiler/xla/service/gpu/gemm_thunk.cc:176] Check failed: stream->parent()->GetBlasGemmAlgorithms(&algorithms) 
Aborted (core dumped)
```

Am I doing something incorrect here? TF was certainly working with my GPU (the program above worked) just a few weeks ago. It seems every time a new TF or keras version is released, I install it and get GPU issues. Any help / advice is much appreciated. Thank you

**Comment by ymodak** [comment]
TF 1.14 pre built binary supports cuda 10.0 where as your configuration shows cuda 10.1
` Could not dlopen library 'libcublas.so.10.0'; dlerror: libcublas.so.10.0: cannot open shared object file: No such file or directory`
Please roll back to cuda 10.0

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ktattan** [comment]
@ymodak apologies for the delay in replying. Thank you for clarifying, that has solved my issue
